docs/content/index.md

```diff
@@ -2,60 +2,148 @@
--   **Highly Available** - Druid is used to back SaaS implementations that need to be up all the time. Your data is still available and queryable during system updates. Scale up or down without data loss.
--   **Scalable** - Existing Druid deployments handle billions of events and terabytes of data per day. Druid is designed to be petabyte scale.
+    timestamp             publisher          advertiser  gender  country  click  price
+    2011-01-01T01:01:35Z  bieberfever.com    google.com  Male    USA      0      0.65
+    2011-01-01T01:03:63Z  bieberfever.com    google.com  Male    USA      0      0.62
+    2011-01-01T01:04:51Z  bieberfever.com    google.com  Male    USA      1      0.45
+    2011-01-01T01:00:00Z  ultratrimfast.com  google.com  Female  UK       0      0.87
+    2011-01-01T02:00:00Z  ultratrimfast.com  google.com  Female  UK       0      0.99
+    2011-01-01T02:00:00Z  ultratrimfast.com  google.com  Female  UK       1      1.53
 
+This data set is composed of three distinct components. If you are familiar with OLAP terminology, the following concepts should not be new.
 
-Why Druid?
-----------
+* **Timestamp column**: We treat timestamp separately because all of our queries
+ center around the time axis.
 
-Druid was originally created to resolve query latency issues seen with trying to use Hadoop to power an interactive service. It's especially useful if you are summarizing your data sets and then querying the summarizations. Put your summarizations into Druid and get quick queryability out of a system that you can be confident will scale up as your data volumes increase. Deployments have scaled up to many TBs of data per hour at peak ingested and aggregated in real-time.
+* **Dimension columns**: Here we have four dimensions of publisher, advertiser, gender, and country.
+They each represent an axis of the data that we’ve chosen to slice across.
 
-Druid is a system that you can set up in your organization next to Hadoop. It provides the ability to access your data in an interactive slice-and-dice fashion. It trades off some query flexibility and takes over the storage format in order to provide the speed.
+* **Metric columns**: These are impressions, clicks and revenue. These represent values, usually numeric,
+which are derived from an aggregation operation – such as count, sum, and mean (we also run variance and higher moment calculations).
+Also known as measures in standard OLAP terminology.
 
-We have more details about the general design of the system and why you might want to use it in our [White Paper](http://static.druid.io/docs/druid.pdf) or in our [Design](Design.html) doc.
+Individually, the events are not very interesting, however, summarizations of this type of data can yield many useful insights.
+Druid can optionally choose to summarize this raw data at ingestion time using a process we refer to as "roll-up".
```

Comment:
I'm wondering why you chose to talk in the introduction about optionally rolling up the data at ingestion time. Generally, talking about options is something done in later sections of a document that go more deeply into aspects of the subject.  Is there something about the optional roll-up that is essential to the "druidness" of druid?  I'd have thought not, since it's optional.